hub/core/storage/cachable.py

```diff
@@ -28,7 +28,14 @@ def __setstate__(self, state: Dict[str, Any]):
 
     # sortkeys / indent make the json more human readable
     def tobytes(self) -> bytes:
-        return bytes(json.dumps(self.__getstate__(), sort_keys=True, indent=4), "utf-8")
+        return bytes(
+            json.dumps(
+                {str(k): v for k, v in self.__getstate__().items()},
```

Comment:
```suggestion
                self.__getstate__().copy(),
```

would this work?

Reply:
No. The keys have to be strings. Its not a copy issue.

Reply:
Can't we simply remove the sort_keys=True? Casting keys to strings from int for the case of info (which was the test that was failing) could be problematic.